tests/rules/metavar_call.php

```diff
@@ -0,0 +1,18 @@
+<?php
+function foo() {
+
+    # ruleid:test
```

Comment:
Ok but I don't think it will work for a pattern like

$X == $X

and then the target PHP code is
foo() == FOO()

Can you add a test for that?